QUEST.tsv

```diff
@@ -1407,96 +1408,96 @@ QUEST_20150317_001407	Sincerity of Moze
+QUEST_20150323_001426	I have no strength but I write this for anyone who may pass by this area. {nl}Naktis… It's a funny thing to wish a curse on the Monarch of Curses.
+QUEST_20150323_001427	Naktis spread cursed Tree Root Crystals in the area. {nl}To stop the curse, they must be destroyed.
+QUEST_20150323_001428	Perhaps I shall soon lose my breath. {nl}Pilgrims, may the blessings of the Goddess be with you!
+QUEST_20150323_001429	Please be sure to find the scripture! {nl}One of the pilgrim souls entrapped in here must have it.
 QUEST_20150323_001430	All pilgrims should offer different amounts.
 QUEST_20150323_001431	It's not the right amount.
-QUEST_20150323_001432	Worship the Goddess Statue to receive blessings or move to another area. 
-QUEST_20150323_001433	What a nasty curse. {nl}Now let's see which is stronger. The curse of Naktis or the purifier of the Goddess. {nl}
-QUEST_20150323_001434	Of course I have the purifier. {nl}But I won't give it unless you offer money.
-QUEST_20150323_001435	It could be a broken plaster of the Goddess Statue to get rid of the monsters there. {nl}If I'm right then it must be Naktis who's behind it. {nl}I may be far away but I hear the stories. {nl}
-QUEST_20150323_001436	Naktis is looking for the clue to enter the Cathedral. {nl}If I'm right, not only the Forest of Prayer but all of the Pilgrim's Way must be like that. {nl}If your journey reaches the Cathedral, please get rid of Naktis. {nl}You have to stop the revelation from falling in the hands of demons.
-QUEST_20150323_001437	This place is over. {nl}All Tree Root Crystals are mutated. {nl}
-QUEST_20150323_001438	Some Tree Root Crystals are useless unless burned. {nl}Some even fume the core. {nl}
-QUEST_20150323_001439	We must do whatever it takes to turn it back but… now it's all tiresome…{nl}
-QUEST_20150323_001440	Feels like drowning in deep water. {nl}The Tree Root Crystal is slowly killing me. {nl}
-QUEST_20150323_001441	Must do whatever it takes to destroy it…{nl}Please save me.
-QUEST_20150323_001442	I have no desire. Am I dying now? {nl}Eating and even breathing feels meaningless. {nl}
-QUEST_20150323_001443	But it must go on.{nl}Unless that cursed Tree Root Crystal is destroyed…
-QUEST_20150323_001444	There are things that Witas cares about. {nl}Won't he get his mind back if you show him those? 
+QUEST_20150323_001432	Worship the Goddess Statue to receive blessings or move on to another area.
+QUEST_20150323_001433	What a nasty curse. {nl}Let's see which is stronger. The curse of Naktis or the purifier of the Goddess.
+QUEST_20150323_001434	Of course I have the purifier. {nl}But I won't give it unless you make a money offering.
+QUEST_20150323_001435	It could be a broken plaster of the Goddess Statue so get rid of the monsters there. {nl}If I'm right, then Naktis must be behind it. {nl}I may be far away but I hear the stories.
```

Comment:
Possibly this, but I'd like some input from someone more familiar:
"It could be a piece broken off of the Goddess Statue, so go there and kill some monsters. {n1}If I'm right, then Naktis must be behind it. {n1} I may be far away, but I am aware of the stories."